### ISSUE TITLE
Consider GitHub Actions

### ISSUE THREAD

**Issue by Ailrun** [CUTE-Lang/miniCUTE#143]
GitHub Actions is free for open source projects, and it supports parallelism too,
whereas CircleCI's free plan is quite limited.

(However, CircleCI's first-class docker support has its advantage.)

**Comment by Ailrun** [comment]
I failed to replace CircleCI because of cache size limit.
GitHub Actions has strict <=400MB cache limit, which is almost impossible to satisfy with our dep size (it requires nearly 450MB).

**Comment by Ailrun** [comment]
Ref: https://github.com/actions/cache/issues/6

**Comment by imbsky** [comment]
The cache limit has been bumped to 2GB.

**Comment by Ailrun** [comment]
@imbsky yes, I subscribed the issue, so I know it is increased. Now I am considering it again.

**Comment by imbsky** [comment]
If 450 MB is enough, there is no problem, but if 1GB is needed, the cache can't be used effectively, so there are still many concerns.

**Comment by Ailrun** [comment]
@imbsky Yes, even with 450MB, it cannot save more than 4 snapshots, and that's why I am reluctant now.

**Comment by Ailrun** [comment]
Now it's 5GB, so I'm planning to introduce it again.
